src/2.6.0/wso2apim.js

```diff
@@ -239,6 +239,15 @@ function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
     if (apiDef.cors) {
       wso2ApiDefinition.corsConfiguration = constructCorsConfiguration(apiDef);
     }
+    if (apiDef.subscriberVisibilityRoles) {
+      wso2ApiDefinition.visibleRoles = apiDef.subscriberVisibilityRoles;
+    }
+    if (apiDef.publisherVisibilityControl) {
```

Comment:
can you keep the naming similar between publisher & subscriber, for easy readability?

`publisherVisibilityControl` --> `publisherVisibility` 
`publisherVisibilityControlRoles` --> `publisherVisibilityRoles`